great_expectations/rule_based_profiler/rule_based_profiler.py

```diff
@@ -376,6 +374,20 @@ def run(
 
         return expectation_suite
 
+    def add_rule(self, rule: Rule) -> None:
+        """
+        Add Rule object to existing profiler object by reconciling profiler rules and updating _profiler_config.
+        """
+        rules_dict: dict = {rule.name: rule.to_json_dict()}
+        effective_rules: List[Rule] = self.reconcile_profiler_rules(
+            rules=rules_dict,
+        )
+        updated_rules: Optional[Dict[str, Dict[str, Any]]] = {
```

Comment:
@Shinnnyshinshin we need to add `rule: Rule` (style).  Thanks!

Reply:
actually this `rule` is a parameter not a local variable. Resolved synchronously